week3/src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import WorldCup from '../src/Components/WorldCup';
+
+function App() {
+
+  return (
+      <main>
+        <WorldCup />
+      </main>
+  );
```

Comment:
```suggestion
  return <WorldCup />;
```

`main`  태그를 `WorldCup` 컴포넌트 내의 최상위 태그로 정의해줘도 좋을 것 같은데?! 지금은 `div`로 정의되어 있는 것 같아서!!